docs/scos/dev/guidelines/project-development-guidelines.md

```diff
@@ -16,83 +16,13 @@ redirect_from:
-
-{% endinfo_block %}
-Spryker OS support: High, you can safely take minor and patch releases.
-
-### Spryker OS Module Customization
-
-When specific OOTB Spryker behavior doesn’t fit Project requirements, you can enable the full power of available for your codebase by extending existing business modules.
-
-As it’s quite a substantial change, we need to go deeper and not only extend OOTB Spryker behaviors but also change it, some of the non-API change become dangerous. That’s why a module constraint to a specific minor version is required (using ~ instead of ^).
-
-Consider using the composition design pattern instead of the direct class extensions: it could increase development costs, but also increases vendor support and simplifies minor updates.
-
-{% info_block infoBox "Example" %}
-
-In my Project, Order entity should not be hydrated during the buying process (we are building a vitrine). In this case, I need to create `\Pyz\Zed\Sales\Business\Order\OrderReader`, which will extend the existing `\Spryker\Zed\Sales\Business\Order\OrderReader` and replace the implementation of the `findOrderByIdSalesOrdermethod()`, where I will adjust the hydration calls.
-
-{% endinfo_block %}
-Spryker OS support: Reduced, you can safely take patch releases.
-
-### Spryker OS Module Replacement
-
-When an existing Spryker module provides functionality that doesn’t fit the Project conceptually, there is a possibility to replace a Spryker module completely using the “composer replace” feature.
-
-This option is not available for every module, as sometimes conceptual changes could lead to high development efforts.
-{% info_block infoBox "Example" %}
+Spryker OS exposes codebase Projects, which enables a high level of customization and can satisfy even the most complex Project business requirements.
 
-In my Project, the URL should be built in a completely different concept that Spryker offers. In this case, I need to create a Project module Url, provide an implementation for every API function (Facade, Client, Service, etc.) and replace OOTB module `spryker/url` with `my-super-project/url`. Spryker features based on URL will use Project implementation to process URLs.
+The project development, the team is free to decide what approach to use. Spryker recommends considering **Configuration**, **Plug and Play**, and **Project modules** first to get maximum from the Spryker OS codebase, atomic releases, leverage minimum efforts for the integration of the new features and keeping system up to date.
 
-{% endinfo_block %}
-Spryker OS support: No support.
+Before starting the project it is recommended to check available development strategies and define one for your implementation {ADD LINK TO A DEVELOPMENT STRATEGIES}. 
```

Comment:
Please fill out the {ADD LINK TO A DEVELOPMENT STRATEGIES}. Was not sure about the proper way of setting the links